coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -27,27 +27,36 @@ export async function convertDispenseNotification(
   headers: Hapi.Util.Dictionary<string>,
   logger: pino.Logger
 ): Promise<hl7V3.DispenseNotification> {
-
   const messageId = getMessageId([bundle.identifier], "Bundle.identifier")
 
   const fhirHeader = getMessageHeader(bundle)
   const fhirPatient = getPatientOrNull(bundle)
   const fhirMedicationDispenses = getMedicationDispenses(bundle)
   const fhirFirstMedicationDispense = fhirMedicationDispenses[0]
+  const fhirContainedMedicationRequest = getMedicationDispenseContained<fhir.MedicationRequest>(
+    fhirFirstMedicationDispense,
+    fhirFirstMedicationDispense.authorizingPrescription[0].reference
+  )
   const fhirLineItemIdentifiers = getLineItemIdentifiers(fhirMedicationDispenses)
+  const fhirPractitionerRole = getMedicationDispenseContained<fhir.PractitionerRole>(
```

Comment:
rename to `fhirContainedPractitionerRole`

Reply:
renamed